src/functionBased/components/TodoContainer.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable */
+import React, { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { Route, Routes } from 'react-router-dom';
```

Comment:
* Avoid disabling eslint as this will help you learn how to fix find and fix errors and improve on your code quality